1001-2000/LIT1750LahaMa.xml

```diff
@@ -1,12 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
-type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1750LahaMa">
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1750LahaMa">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
             <title xml:id="t1" xml:lang="gez">ላሐ፡ ማርያም፡</title>
-            <title xml:lang="gez" type="normalized" corresp="#t1">Lāḥā Māryām</title>
-            <title xml:lang="gez" type="normalized" xml:id="t2">Lāḥā Dǝngǝl</title>
+            <title xml:lang="gez" type="normalized" corresp="#t1">Lāḥa Māryām</title>
+            <title xml:lang="en" type="normalized" corresp="#t1">Lamentation of Mary</title>
+            <title xml:lang="gez" xml:id="t2">ላሐ፡ ድንግል፡</title>
+            <title xml:lang="gez" type="normalized" corresp="#2">Lāḥa dǝngǝl</title>
+            <title xml:lang="en" type="normalized" corresp="#t2">Lamentation of the Virgin</title>
+            <title xml:lang="gez" xml:id="t3">ድርሳን፡ በእንተ፡ ክብራ፡ ለድንግል፡</title>
+            <title xml:lang="gez" corresp="#t3" type="normalized">Dǝrsān ... baʾǝnta kǝbrā la-dǝngǝl</title>
```

Comment:
What for are the dots ... necessary?

Reply:
ዘደረሰ፡ አባ፡ ህርያቆስ፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ ብንሀሳ፡ - what would you suggest? better to include it, or to have ... also in Fidal and the translation?

Reply:
I would take it over ደረሰ፡ አባ፡ ህርያቆስ፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ ብህንሳ፡ (!!!) በእንተ፡ ክብራ፡ ለድንግል፡ , for a homily, it is fine to show explicitely the purported author and the subject together.